spec/services/message_employee_matcher_spec.rb

```diff
@@ -18,6 +18,26 @@
       Timecop.return
     end
 
+    it "matches messages to users if the user's time has advanced past the scheduled message send at time but hasn't been sent yet" do
+      Timecop.freeze(Time.current)
+
+      days_after_start = 3
+      scheduled_message_time = Time.current.in_time_zone("Eastern Time (US & Canada)")
+      scheduled_message_time -= 1800
+      puts Time.current
```

Comment:
I do it allll the time - I think there's a code climate thing you can set up to check for this. we really need to get this project set up on CC....